test/cmd/newapp.sh

```diff
@@ -97,7 +97,7 @@ os::cmd::expect_success 'oc delete all -l app=ruby'
 
 # check new-build
 os::cmd::expect_failure_and_text 'oc new-build mysql -o yaml' 'you must specify at least one source repository URL'
-os::cmd::expect_success_and_text 'oc new-build mysql --binary -o yaml' 'type: Binary'
+os::cmd::expect_success_and_text 'oc new-build mysql --binary -o yaml --to mysql:bin' 'type: Binary'
```

Comment:
Fixing this test, since `oc new-build mysql --binary -o yaml` creates an infinite build loop.